ocfweb/docs/docs/services/lab/lab-reservation-policy.md

```diff
@@ -0,0 +1,55 @@
+[[!meta title="Lab reservation policy"]]
+
+Registered Student Organizations of UC Berkeley are able to reserve the OCF lab
+space for a maximum of 2 hours at a time during the OCF’s regular operating
+hours. Those who reserve the OCF lab space are able to access the 28 computers
+within the lab (as well as additional seating), in addition to the main TV
+attached to the back wall. The OCF can additionally provide a whiteboard and
+printing (pursuant to printing quotas) to student groups who utilize the lab
+space. Should the lab be reserved, the option to fully close the lab to the
+outside population (excluding OCF front-desk staff) is readily available.
```

Comment:
"readily" available? Why not "available if necessary"?

Reply:
I think we covered most of these things in the comments on the Drive doc

Reply:
That was changed. I would hold off on reviewing wording until @edricx puts the new one in :)